packages/synapse-react-client/src/components/CertificationQuiz/CertificationQuiz.integration.test.tsx

```diff
@@ -4,14 +4,45 @@ import React from 'react'
 import CertificationQuiz from './CertificationQuiz'
 import * as ToastMessage from '../ToastMessage/ToastMessage'
 import { createWrapper } from '../../testutils/TestingLibraryUtils'
+import {
+  mockQuiz,
+  mockPassingRecordPassed,
+  mockPassingRecordFailed,
+} from '../../mocks/mockCertificationQuiz'
+import {
+  useGetPassingRecord,
+  usePostCertifiedUserTestResponse,
+} from '../../synapse-queries/user/useCertificationQuiz'
+import { getUseMutationMock } from '../../testutils/ReactQueryMockUtils'
+import { PassingRecord, QuizResponse } from '@sage-bionetworks/synapse-types'
+import { SynapseClientError } from '../../utils'
 import {
   BackendDestinationEnum,
   getEndpoint,
 } from '../../utils/functions/getEndpoint'
 import { rest, server } from '../../mocks/msw/server'
-import { mockQuiz, mockPassingRecord } from '../../mocks/mockCertificationQuiz'
+import { useGetCurrentUserBundle } from '../../synapse-queries/user/useUserBundle'
+import { mockUserBundle } from '../../mocks/user/mock_user_profile'
+import { formatDate } from '../../utils/functions/DateFormatter'
+import dayjs from 'dayjs'
 
 window.open = jest.fn()
+jest.mock('../../synapse-queries/user/useCertificationQuiz', () => {
+  return {
+    usePostCertifiedUserTestResponse: jest.fn(),
+    useGetPassingRecord: jest.fn(),
+  }
+})
+const mockUsePostCertifiedUserTestResponse =
+  usePostCertifiedUserTestResponse as jest.Mock
+const mockUseGetPassingRecord = useGetPassingRecord as jest.Mock
```

Comment:
Can get extra type safety, type hinting by doing this
```suggestion
const mockUsePostCertifiedUserTestResponse = jest.mocked(usePostCertifiedUserTestResponse)
const mockUseGetPassingRecord = jest.mocked(useGetPassingRecord)
```